views/whereabouts.njk

```diff
@@ -12,13 +16,76 @@
       href: "/search-prisoner-whereabouts"
     },
     {
-      text: "Dashboard"
+      text: "Statistics: attendance reasons"
     }
   ]
 }) 
 }}
 {% endblock %}
 
 {% block content %}
-Whereabouts Dashboard goes here...
+
+{% set urlParams = '/whereabouts?agencyId=' + caseLoadId + '&period=AM&date=2019-08-23'%}
+
+<h1 class="govuk-heading-l mainHeading">Statistics: attendance reasons</h1>
+<form action="/whereabouts">
+<div class="govuk-grid-row">
+  <input type="hidden" name="agencyId" value="{{caseLoadId}}" />
+  <div class="govuk-grid-column-one-quarter">
+    {% include "./components/datePicker.njk" %}
+  </div>
+  <div class="govuk-grid-column-one-quarter">
+    {% include "./components/periodSelect.njk" %}
+  </div>
+</div>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-quarter">
+    {{
+        govukButton({
+          type: submit,
+          text: 'Update',
+          classes: 'govuk-button'
+        })
+      }}
+  </div>
+</div>
+</form>
+<hr>
+<h2 class="govuk-heading-m">Paid reasons</h2>
+<div class="govuk-grid-row govuk-section-break--m">
+  <div class="govuk-grid-column-one-quarter">
+    <p class="govuk-body">Attended</p>
+    <span class="govuk-fieldset__legend--xl">{{data.attended}}</span>
```

Comment:
`govuk-fieldset__legend--xl` should be used only with `govuk-fieldset__legend` and should probably be on a legend if that's the appropriate element.